testdata/libclangtest/code/a.cc

```diff
@@ -26,4 +27,10 @@ void allReqsCovered() {}
 // @llr REQ-TEST-SWL-3
 using MyType = int;
 
+// @llr REQ-TEST-SWL-3
```

Comment:
Add concept without llr too

Reply:
Sure, although the test does not check the parsing of any existing requirements, just that an optional tag is identified, that's tested elsewhere